index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>'Volker Jacobsen'</title>
+    <img src="https://cdn.pixabay.com/photo/2016/10/20/20/58/banner-1756639_1280.jpg" alt="Headerbild" >
+</head>
+<body>
+    <header>
+        <h1>Volker Jacobsen</h1>
+        <nav id="nav">
+            <a href="#footer">Contact</a>
+            <a href="Biography">Biography</a>
+        </nav>
+        <img src="https://cdn.pixabay.com/photo/2016/10/20/20/58/banner-1756639_1280.jpg" alt="Headerbild" >
+    </header>
+    <main>
+        <section>
+            <p> Hi, my name is Volker Jacobsen. Welcome to my personal website. I am a Senior project manager
+                with more than 20 years of professional experience. 
+                Currently I am attending a Web development bootcamp @ neuefische GmbH. 
+            </p>
+        </section>
+        <section id="buttons">
+            <h3>Follow me on:</h3>
+            <a href="https://github.com/VolkerJacobsen" target="_parent"><button type="image"> <img src="/img/DelftStack/logo.png" style="border: double;" height ="225" width="400" /></button></a>
+            <a href="https://www.linkedin.com/in/volker-jacobsen-b2893421b/" target="_parent"><button type="image"> <img src="/img/DelftStack/logo.png" style="border: double;" height ="225" width="400" /></button></a>
+            <a href="https://codepen.io/volker-jacobsen" target="_parent"><button type="image"> <img src="/img/DelftStack/logo.png" style="border: double;" height ="225" width="400" /></button></a>
+        </section>
```

Comment:
The images of the buttons don't load, maybe you didn't upload the folder containing the images?